buildkite/build_project.sh

```diff
@@ -147,6 +147,12 @@ case "$REPO_FULL_NAME" in
         CONFIG=select ./travis-ci.sh
         ;;
 
+    dlang/ci)
+        echo "Check that the PR commit has been merged into the target branch"
+        echo "Current commit: $(git describe)"
+        [[ "$(git log --format=%B -n 1)" =~ Merge[[:space:]]${BUILDKITE_COMMIT:-invalid} ]]
```

Comment:
A simple test, s.t. we don't accidentally break this in the future (and also actually can test this here).